INSTALLING.md

```diff
@@ -26,12 +26,13 @@ Note the current compatability matrix:
 
 ```sh
 sudo apt update
-sudo apt install software-properties-common=0.99.9.12 build-essential=12.8ubuntu1.1 python3.8-venv=3.8.10-0ubuntu1~20.04.9 libyaml-cpp-dev=0.6.2-4ubuntu1 libhwloc-dev graphviz
+sudo apt install software-properties-common=0.99.9.12 build-essential=12.8ubuntu1.1 python3.8-venv=3.8.10-0ubuntu1~20.04.9 libhwloc-dev graphviz
 
-# Install Clang-17 for C++20 support!!
```

Comment:
I think because we now depend on `libc++`, we should probably merge these two code sections so it's continous.

Reply:
How do you mean?

Reply:
Actually, never mind my original comment.

Can you please get rid of the `Install clang-17...` comment?

I'm concerned users may not do it, when it's in fact necessary for `libc++`.

Reply:
Will do!